internal/storage/postgres/seed/99_demo_data.sql

```diff
@@ -35,6 +35,9 @@ INSERT INTO sensors (id, status, latitude, longitude, geometry)
 VALUES
     ('sensor-4', 'online', 54.82078826498143, 9.489684366114483, ST_SetSRID(ST_MakePoint(54.82078826498143, 9.489684366114483), 4326));
 
+INSERT INTO sensors (id, status, latitude, longitude, geometry)
+VALUES
+    ('tree-sensor', 'online', 54.774932, 9.450000, ST_SetSRID(ST_MakePoint(54.774932, 9.450000), 4326));
```

Comment:
I don't know exactly what the purpose of this is, but ok

Reply:
It was a test sensor to check if the ID "tree-sensor" from TTN exists or not.